packages/frontend-v2/components/AddCourseModal/SearchResult.tsx

```diff
@@ -1,7 +1,7 @@
 import { SmallAddIcon } from "@chakra-ui/icons";
 import { Flex, Box, Heading, IconButton, Text } from "@chakra-ui/react";
 import { ScheduleCourse2 } from "@graduate/common";
-import { getCourseDisplayString } from "../../utils/course/getCourseDisplayString";
```

Comment:
a couple of changes having to do with importing directly from "utils" rather than the actual file